src/common/components/body-class-modifier.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { DocumentManipulator } from 'common/document-manipulator';
+import * as React from 'react';
+
+export type BodyClassModifierProps = {
+    classNames: string[];
+    documentManipulator: DocumentManipulator;
```

Comment:
should this be a dep instead of a prop?

Reply:
I don't feel strongly about it either way, switched